.rubocop_todo.yml

```diff
@@ -1,29 +1,108 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2023-12-19 10:44:38 UTC using RuboCop version 1.48.1.
+# on 2024-06-13 18:25:00 UTC using RuboCop version 1.64.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowForAlignment, AllowBeforeTrailingComments, ForceEqualSignAlignment.
+Layout/ExtraSpacing:
+  Exclude:
+    - 'spec/puppet/resource_api/transport_schema_def_spec.rb'
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
```

Comment:
Might be good to try and do the safe autocorrects

Reply:
Hi @david22swan.  I thought about getting this across the line with a green CI and then doing further autocorrects in another PR.  The autocorrects I included in this PR were necessary to keep CI green for the `jruby` environment tests.